java-checks-test-sources/src/main/java/checks/AnonymousClassShouldBeLambdaCheck.java

```diff
@@ -89,19 +92,20 @@ public String handle() {
       @Override
       public String handle() {
         return "";
-      }; // this empty statement should not be counted!
+      } // this empty statement should not be counted!
```

Comment:
This change does not make sense to me, we should either add the empty statement or remove the comment.

Reply:
indeed, reverted.